src/symbolic_regression/symbolic_regression.jl

```diff
@@ -0,0 +1,125 @@
+using SymbolicRegression
+using DataDrivenDiffEq
+
+
+"""
+$(TYPEDEF)
+
+Options for using SymbolicRegression.jl within the `solve` function.
+Automatically creates [`Options`](https://astroautomata.com/SymbolicRegression.jl/stable/api/#Options) with the given specification.
+Sorts the operators stored in `functions` into unary and binary operators on conversion.
+
+# Fields
+$(FIELDS)
+
+"""
+struct EQSearch <: AbstractSymbolicRegression
+    "Operators used for symbolic regression"
+    functions::AbstractVector{Function}
+    "Additionally keyworded arguments passed to SymbolicRegression.Options"
+    kwargs
+end
+
+function EQSearch(functions = Function[/, +, *, exp, cos]; kwargs...)
+    return EQSearch(functions, kwargs)
+end
+
+function to_options(x::EQSearch)
+    binops = Tuple([fi for fi in x.functions if is_binary(fi)])
+    unaops = Tuple([fi for fi in x.functions if is_unary(fi)])
+    fnames = fieldnames(SymbolicRegression.Options)
+    ks = Dict(
+        [k => v for (k,v) in x.kwargs if k ∈ fnames]
+    )
+    return SymbolicRegression.Options(;
+        binary_operators = binops, unary_operators = unaops,
+        ks...
+    )
+end
+
+function DiffEqBase.solve(prob::AbstractDataDrivenProblem, alg::EQSearch;
+    max_iter::Int = 10,
+    weights = nothing,
+    numprocs = nothing, procs = nothing,
+    multithreading = false,
```

Comment:
I would set this default to `true`, and I think nothing else needs to be changed.